src/identity/apis/auth.ts

```diff
@@ -96,6 +103,42 @@ export const fetchIdentity = async () => {
   return fetchQuartzMe()
 }
 
+const identityRetryDelay = 30000 // 30 seconds
```

Comment:
Should we flag for services and e-commerce that these are the delay timings we're using so they can weigh in? (starting at thirty, incrementing delay by 30s each time)?

Reply:
I think that's a good idea, but I'll flag them when it comes time to integrate this change into the application